lib/pwa.js

```diff
@@ -323,10 +324,14 @@ exports.fetchManifest = function(pwa) {
  * @return {Pwa} the updated PWA
  */
 exports._save = function(pwa) {
+  let created = true;
```

Comment:
Not sure I like storing the created state in a variable in the outer scope. Maybe return a wrapper object inside the promise instead, e.g.:

```
return {
  pwa: pwa,
  created: true
};
```